mantis-rxcontrol/src/main/java/io/mantisrx/control/clutch/ExperimentalClutchConfigurator.java

```diff
@@ -134,7 +134,7 @@ private static void logSketchSummary(DoublesSketch sketch) {
     private static void logStatsSummary(DescriptiveStatistics stat, String prefix) {
         log.info("{} RPS Sketch Quantiles -- Min: {}, 25th: {}, 50th: {}, 75th: {}, 99th: {}, Max: {}",
                 prefix,
-                stat.getPercentile(0),
+                stat.getMin(),
```

Comment:
can you add a desc to explain the context and root cause?

Reply:
Will do.